.github/CODEOWNERS

```diff
@@ -0,0 +1,23 @@
+# Copied files:
+
+.github/dco.yml @IBM/cloudant-sdks
+scripts/setup_couch.sh @IBM/cloudant-sdks
+scripts/setup_wiremock.sh @IBM/cloudant-sdks
+stubs/mappings.json @IBM/cloudant-sdks
+KNOWN_ISSUES.md @IBM/cloudant-sdks
+Jenkinsfile @IBM/cloudant-sdks
+
+# Generated code files:
+
+src/main/java/com/ibm/cloud/cloudant/v1/* @IBM/cloudant-sdks
+src/test/java/com/ibm/cloud/cloudant/v1/* @IBM/cloudant-sdks
+
+# Bump version files:
+
+.bumpversion.cfg @IBM/cloudant-sdks
+# README.md @IBM/cloudant-sdks
```

Comment:
I suggest we remove the commented lines.  I think it may add confusion.

Reply:
As we agreed with @ricellis offline, I will add these files (except the README).